machine_management/applying-autoscaling.adoc

```diff
@@ -9,6 +9,12 @@ Applying autoscaling to an {product-title} cluster involves deploying a
 ClusterAutoscaler and then deploying MachineAutoscalers for each Machine type
 in your cluster.
 
+[IMPORTANT]
+====
+You can configure the ClusterAutoscaler only in clusters that use
```

Comment:
Is this doc generic for upi and ipi? If so may be mention you can configure  autoscaling in UPI if the machine API is operational

Reply:
We don't currently document using the machineAPI to create machines in a UPI cluster. There was some disagreement about whether or not to include this method for AWS. If it's something that we want customers to use, please let me know so that I can get an issue to add that doc in our queue.